### PR TITLE
IDENTITY-3184

### PR DESCRIPTION
**Pull request by wattale**

### CODE REVIEW

File: modules/samples/identity-mgt/InfoRecoverySample/src/main/java/org/wso2/sample/inforecovery/controller/VerificationController.java [hunk 1]
```diff
@@ -62,9 +62,11 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Servl
         HttpSession session = req.getSession(false);
         String confirmation = (String) session.getAttribute("confirmation");
         String viewPage = null;
-
+        String userstoredomain = req.getParameter("userstoredomain");
         String username = req.getParameter("username");
-
+        if (!("PRIMARY".equalsIgnoreCase(userstoredomain)) && userstoredomain != null) {
```
Comment (GayanM) [inline modules/samples/identity-mgt/InfoRecoverySample/src/main/java/org/wso2/sample/inforecovery/controller/VerificationController.java]: Use user core util methods and user core constants

File: modules/samples/identity-mgt/InfoRecoverySample/src/main/java/org/wso2/sample/inforecovery/controller/VerificationController.java [hunk 2]
```diff
@@ -62,9 +62,11 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Servl
         HttpSession session = req.getSession(false);
         String confirmation = (String) session.getAttribute("confirmation");
         String viewPage = null;
-
+        String userstoredomain = req.getParameter("userstoredomain");
```
Comment (GayanM) [inline modules/samples/identity-mgt/InfoRecoverySample/src/main/java/org/wso2/sample/inforecovery/controller/VerificationController.java]: check getSafeText

File: modules/samples/identity-mgt/InfoRecoverySample/src/main/webapp/assets/css/landing-page.css [hunk 3]
```diff
@@ -1,158 +1,158 @@
...
-        padding: 0;
-    }
-
-    ul.banner-social-buttons > li:last-child {
-        margin-bottom: 0;
-    }
-}
-
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (GayanM) [inline modules/samples/identity-mgt/InfoRecoverySample/src/main/webapp/assets/css/landing-page.css]: check license year

Comment (wattale) [inline modules/samples/identity-mgt/InfoRecoverySample/src/main/webapp/assets/css/landing-page.css]: Fixed in the PR : https://github.com/wso2/product-is/pull/182

File: modules/samples/identity-mgt/InfoRecoverySample/src/main/webapp/css/cart-styles.css [hunk 4]
```diff
@@ -1,248 +1,248 @@
...
-
-.checkout-wrapper {
-    padding-top: 10px;
-    text-align: right;
-}
-
-.cartData h1 {
-    color: #fff;
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (prabathabey) [inline modules/samples/identity-mgt/InfoRecoverySample/src/main/webapp/css/cart-styles.css]: Fix year of the license.

Comment (wattale) [inline modules/samples/identity-mgt/InfoRecoverySample/src/main/webapp/css/cart-styles.css]: Fixed in the PR : https://github.com/wso2/product-is/pull/182

**Pull request merged**